docs/crio.conf.5.md

```diff
@@ -290,6 +293,8 @@ the container runtime configuration.
 **enable_criu_support**=false
   Enable CRIU integration, requires that the criu binary is available in $PATH. (default: false)
 
+
+
```

Comment:
this whitespace addition seems unnecessary

Reply:
done